package.json

```diff
@@ -64,43 +64,43 @@
     "upgrade:manual": "sh ./scripts/monorepo-npm-upgrade.sh"
   },
   "devDependencies": {
-    "@babel/core": "^7.23.9",
-    "@babel/eslint-parser": "^7.23.9",
-    "@babel/preset-react": "^7.17.12",
-    "@commitlint/cli": "^18.6.0",
-    "@commitlint/config-conventional": "^18.6.0",
-    "@playwright/test": "^1.36.2",
+    "@babel/core": "^7.24.5",
+    "@babel/eslint-parser": "^7.24.5",
+    "@babel/preset-react": "^7.24.1",
+    "@commitlint/cli": "^19.3.0",
+    "@commitlint/config-conventional": "^19.2.2",
+    "@playwright/test": "^1.44.0",
     "@testing-library/dom": "^8.11.4",
     "@testing-library/react": "^14.0.0",
     "@testing-library/react-hooks": "^8.0.1",
     "@testing-library/user-event": "^14.4.3",
-    "@typescript-eslint/eslint-plugin": "^6.21.0",
-    "@typescript-eslint/parser": "^6.21.0",
-    "accessibility-checker": "^3.1.65",
+    "@typescript-eslint/eslint-plugin": "^7.9.0",
+    "@typescript-eslint/parser": "^7.9.0",
+    "accessibility-checker": "^3.1.71",
     "cheerio": "^1.0.0-rc.12",
     "copyfiles": "^2.4.1",
-    "cspell": "^8.3.2",
-    "eslint": "^8.56.0",
+    "cspell": "^8.8.1",
+    "eslint": "^9.2.0",
```

Comment:
`eslint-config-carbon` is still using v8, which will probably conflict with this upgrade. At a glance, it looks like `.eslintignore` is no longer supported in v9, and must be included in the config file. Suggest leaving this at v8 until it's necessary to upgrade.